src/ucm/mmap/install.c

```diff
@@ -102,18 +102,11 @@ void ucm_fire_mmap_events(int events)
     }
 }
 
-/* Called with lock held */
-static ucs_status_t ucm_mmap_test(int events)
+ucs_status_t ucm_test_events(int events)
```

Comment:
don't we need to test also malloc/free?

Reply:
don't think so - malloc/free are used for internal implementation, not an event